packages/zosfiles/src/methods/copy/Copy.ts

```diff
@@ -71,6 +72,15 @@ export class Copy {
         if (newDataSet) {
             await Create.dataSetLike(session, toDataSetName, options["from-dataset"].dsn);
         }
+        else if(safeReplace) {
+            if (options.promptFn != null) {
```

Comment:
You added 'promptFn' to ICopyDatasetOptions.

The Copy.dataSetCrossLPAR function's 'options' parameter is of type ICrossLparCopyDatasetOptions, which inherits from ICopyDatasetOptions and adds its own 'promptFn' attribute.

Should we remove 'promptFn' from ICrossLparCopyDatasetOptions and confirm that the Copy.dataSetCrossLPAR function continues to work correctly?